src/test/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImportIntegrationTest.java

```diff
@@ -52,16 +51,23 @@
     private static final List<String> LOCAL_GVCFS = Arrays.asList(HG_00096, HG_00268, NA_19625);
     private static final String GENOMICSDB_TEST_DIR = toolsTestDir + "GenomicsDBImport/";
     private static final String COMBINEGVCFS_TEST_DIR = toolsTestDir + "walkers/CombineGVCFs/";
-
     private static final String COMBINED = largeFileTestDir + "gvcfs/combined.gatk3.7.g.vcf.gz";
+    private static final String COMBINED_MULTI_INTERVAL = largeFileTestDir + "gvcfs/combined_multi_interval.gatk3.7.g.vcf.gz";
     private static final String COMBINED_WITHSPACES = largeFileTestDir + "gvcfs/combined.gatk3.7.smaller_interval.g.vcf";
-    private static final SimpleInterval INTERVAL = new SimpleInterval("chr20", 17960187, 17981445);
-    private static final SimpleInterval INTERVAL_3736 = new SimpleInterval("chr6",130365070,146544250);
-    private static final SimpleInterval INTERVAL_NONDIPLOID = new SimpleInterval("20", 10000000, 10100000);
-    private static final SimpleInterval SMALLER_INTERVAL = new SimpleInterval("chr20", 17960187, 17961973);
+    private static final ArrayList<SimpleInterval> INTERVAL =
+            new ArrayList<SimpleInterval>(Arrays.asList(new SimpleInterval("chr20", 17960187, 17981445)));
+    private static final ArrayList<SimpleInterval> MULTIPLE_INTERVALS = new ArrayList<SimpleInterval>(Arrays.asList(
+        new SimpleInterval("chr20", 17960187, 17970000),
+        new SimpleInterval("chr20", 17970001, 17980000),
+        new SimpleInterval("chr20", 17980001, 17981445)
```

Comment:
When specifying multiple intervals, are they required to be contiguous? If not, could you please add a separate test case involving non-contiguous intervals as well?

Reply:
Also add a third test case involving multiple intervals from different contigs.

Reply:
Intervals need not be contiguous